core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/RangerVirtualGroupService.java

```diff
@@ -33,8 +34,13 @@
     public String getRangerVirtualGroup(Stack stack) {
         Optional<LdapView> ldapView = ldapConfigService.get(stack.getEnvironmentCrn(), stack.getName());
         String virtualGroupsEnvironmentCrn = environmentConfigProvider.getParentEnvironmentCrn(stack.getEnvironmentCrn());
-        String adminGroup = ldapView.orElseThrow(() -> new CloudbreakServiceException("Ranger admin group not found."))
-                .getAdminGroup();
+        String adminGroup;
+        if (CloudPlatform.MOCK.equalsIgnoreCase(stack.getCloudPlatform())) {
```

Comment:
Any way you could move this check above everything else, like at the beginning of the function? Seems kind of pointless to run other logic if this statement will discard their results.

Reply:
Addressed the comment.